src/ffq_api/app.py

```diff
@@ -38,47 +38,63 @@ def read_root(request: Request):
 @app.get("/v1alpha1/{accession_str}")
 def read_item(
     accession_str: str,
-    search_type: Union[Search_Types, None] = None,
+    search_type: Union[SearchTypes, None] = None,
     ftp: bool = False,
     aws: bool = False,
     gcp: bool = False,
     ncbi: bool = False,
```

Comment:
If these are mutually exclusive, could just have it as a single variable with a string response (eg. `link_type=aws`).